src/goto-symex/goto_symex_state.cpp

```diff
@@ -51,11 +54,9 @@ void goto_symex_statet::level0t::operator()(
 
   const symbolt *s;
 
-  if(ns.lookup(obj_identifier, s))
-  {
-    std::cerr << "level0: failed to find " << obj_identifier << '\n';
-    abort();
-  }
```

Comment:
We had abort() calls hidden in the depths of the system :-\  Scary...

Reply:
Good call - `git grep -w 'abort()'` should be reviewed, possibly as part of this PR?

Reply:
#1902.